src/commands/SinceCommand.ts

```diff
@@ -0,0 +1,240 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Mjolnir } from "../Mjolnir";
+import { LogLevel, LogService, RichReply } from "matrix-bot-sdk";
+import { htmlEscape, parseDuration } from "../utils";
+import { ParseEntry } from "shell-quote";
+import { HumanizeDurationLanguage, HumanizeDuration } from "humanize-duration-ts";
+
+const HUMANIZE_LAG_SERVICE: HumanizeDurationLanguage = new HumanizeDurationLanguage();
+const HUMANIZER: HumanizeDuration = new HumanizeDuration(HUMANIZE_LAG_SERVICE);
+
+enum Action {
```

Comment:
There doesn't seem to be any feedback at all in the command room when the command sent is invalid?

![image](https://user-images.githubusercontent.com/50846879/157036743-7e30a89a-e882-44c1-808b-0d173c628a56.png)

Reply:
I'm not sure the help is very good either